### DISCUSSION TITLE
DifferentExample

### DISCUSSION THREAD

**Discussion by raymadigan** [Dlloydev/sTune#12] [Q&A]
I want to check out your library and have a slightly different environment:

I have an Adafruit_MAX31865 instead of Max31856, I assume I can just pass this controller in, I will look closer.
I have a relay with two pins, one on and one off.  
I'm not sure what the drdyPin is, I will sort that out next;

Do you have a problem if I copy the example and make it work with these two modifications?  I will be able to tell you the changes I made or just give you the result after I have it worked out.

**Comment by Dlloydev** [comment]
No problem, use whatever you need!

I think there'll be more than a few modifications, but not too difficult. You'll need to install their library then I'd probably start by getting their [example](https://github.com/adafruit/Adafruit_MAX31865/blob/master/examples/max31865/max31865.ino) to work. I took a quick look, note that it defaults to software SPI and the PT100 RTD sensor. You may need to change to RNOMINAL to 1000.0 if you have the PT1000 sensor.

I would suggest verifying that everything works with software SPI first, then test the hardware SPI settings and pin configuration that matches or closely matches what is used with the MAX31856.

I've used the drdyPin on the 31856 to speed up reads (they describe this on [page 6 here](https://cdn-learn.adafruit.com/downloads/pdf/adafruit-max31856-thermocouple-amplifier.pdf)). On the PCB its labelled DRDY.

Looks like this same feature is also available on your [max31865](https://learn.adafruit.com/adafruit-max31865-rtd-pt100-amplifier/pinouts) where the breakout pin is labelled RDY.

It would be interesting to hear how this works out with your setup.

**Reply by raymadigan** [reply]
Thanks for your info, I have very much work to do, I am using it for a gas solenoid to control a propane meat smoking device.

It turns out the adafruit driver does not use the RDY pin so for now I can just ignore it.  If I need more speed, I doubt it for this application, I will have to write my own driver.  I have been logging the temperature in the device for a while, I'm sure I understand how the thermometer works.

I appreciate your response, I will get to start working on it beginning of next week.  Once I have it all sorted out, I will get back to you.
